src/blog/2025-03-15-oxlint-beta.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Oxlint Beta
+outline: deep
+authors:
+  - boshen
+  - camchenry
+---
+
+<AppBlogPostHeader />
+
+We are thrilled to announce that oxlint is now in beta release, after more than a year of development by the community!
+
+This milestone signifies feature completeness and stability.
+
+## How to Use
+
+At this stage, oxlint can be used to fully replace eslint in small to medium projects.
+
+For larger projects, our advice is to turn off eslint rules via the eslint plugin [eslint-plugin-oxlint](https://www.npmjs.com/package/eslint-plugin-oxlint), and run oxlint before ESLint in your lint-staged or CI setup.
+
+To test oxlint in your JavaScript / TypeScript codebase, execute the following command at the root directory of your repository:
+
+::: code-group
+
+```sh [npm]
+$ npx oxlint@latest
+```
+
+```sh [pnpm]
+$ pnpm dlx oxlint@latest
+```
+
+```sh [yarn]
+$ yarn dlx oxlint@latest
+```
+
+```sh [bun]
+$ bunx oxlint@latest
+```
+
+```sh [deno]
+$ deno run npm:oxlint@latest
+```
+
+:::
+
+Alternatively, refer to the [installation guide](/docs/guide/usage/linter) for detailed instructions.
+
+## New Features since General Availability
```

Comment:
Love it! 🫶 
Do we want to tell about IDE Plugins too? 
We have `vscode`, `zed`, `coc.nvim` and `intellij` (only available in preview channel).

Reply:
I mentioned this as an area of future improvement, as I don't think we've truly invested a lot of time into the editor experience yet. But I think it's worth mentioning for sure.